app/views/admin/neighbourhoods/edit.html.erb

```diff
@@ -2,9 +2,19 @@
   const currentInfo = <%= @neighbourhood.to_json.html_safe %>
 </script>
 
-<div id="js-ward-picker">
-  <%= render_component "admin_edit", model: :neighbourhood, title: @neighbourhood.name %>
-  <% if policy(@neighbourhood).new? %>
+<%
+=begin%>
+ <div id="js-ward-picker"> 
+<%
+=end%>
+  <%= render_component "admin_edit",
+         model: :neighbourhood, 
+         title: @neighbourhood.name 
+  %>
+  <%
+=begin%>
+  NOTE: removing this for now!
```

Comment:
I don't understand this syntax. Is it a block comment?

Should we simply delete rather than 'remove for now'?

Reply:
Yep, and ironically I didn't know about it until VS code did it for me :) it's because if you html comment then the ruby will still run basically. Will delete.